### PR TITLE
How To Enable SFTP Without Shell Access on CentOS 7

### PR DESCRIPTION
**Pull request by 1817666**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: MohammadHossein Fakhraei eng.fakhraei@gmail.com

### CODE REVIEW

File: tutorials/enable-sftp-without-shell-access-on-centos-7/01.en.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/enable-sftp-without-shell-access-on-centos-7"
+slug: "enable-sftp-without-shell-access-on-centos-7"
+date: "2019-05-21"
+title: "How To Enable SFTP Without Shell Access on CentOS 7"
+short_description: "This tutorial explains how to enable SFTP without Shell Access on CentOS 7"
```
[the remark this answers is not in the document]
Comment (1817666) [reply tutorials/enable-sftp-without-shell-access-on-centos-7/01.en.md]: I think the title "How To Configure SFTP Without Providing Shell Access on CentOS 7" is good for this tutorial.
What is your opinion? Do you have a better offer?

[the remark this answers is not in the document]
Comment (n-se) [reply tutorials/enable-sftp-without-shell-access-on-centos-7/01.en.md]: Maybe "How to configure SFTP without allowing/granting/whatever/ SSH access on CentOS 7"? We would have to find a title that couldn't be confused with enabling SFTP without SSH access.

[the remark this answers is not in the document]
Comment (1817666) [reply tutorials/enable-sftp-without-shell-access-on-centos-7/01.en.md]: The title edited to "How To Configure SFTP Without Allowing SSH Access on CentOS 7".

[the remark this answers is not in the document]
Comment (1817666) [reply tutorials/enable-sftp-without-shell-access-on-centos-7/01.en.md]: @n-se The title is OK? or You have another offer?

**Pull request merged**